src/controllers/accountController.ts

```diff
@@ -0,0 +1,28 @@
+import { Controller, Get, Param, HttpStatus, Body } from '@nestjs/common';
+import { AccountService } from 'src/services/accountService';
+import { AccountDTO } from 'src/models/modelAccount';
+
+@Controller('account')
+export class AccountController {
+  constructor(private readonly accountService: AccountService) { }
+
+  @Get()
+  async getAllAccounts() {
+    const accounts = await this.accountService.findAllAccounts();
+    return {
```

Comment:
Gostei bastante de ver que seus controllers apenas chama a classe service e que a lógica está no service.
Vi que você está usando um padrão de retorno para todas requests, com status code, mensagem e o dado propriamente dito. Além disso, quando tem um erro, retorna uma exception com o status code correspondente, como Not Found ou Forbidden! Isso é uma ótima prática de programação pois você foi além do caminho feliz e está lidando com as possibilidades de erro tbm 😍